devs/docs/tests.rst

```diff
@@ -19,6 +19,11 @@ For example::
 
     $ ./mvnw test -pl server
 
+To run tests for `server` module and build any dependency module beforehand, but
+skip running tests on those modules, you can run::
+
+    $ ./mvnw test -P serverTest -am
```

Comment:
It seem like by using this new profile, all modules will be included (compiled), so the `-am` has not effect and is such not needed.

Reply:
true, haven't tried it without `-am`, thx!